src/pages/standup/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { FC, useState } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+
+import { standupUpdateType } from '@/interfaces/standup.type';
+import { FormatDate } from '@/utils/FormatDate';
+import { LOGIN_URL } from '@/constants/url';
+import useAuthenticated from '@/hooks/useAuthenticated';
+import StandUpContainer from '@/components/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+import { skipToken } from '@reduxjs/toolkit/dist/query';
+
+const StandUp: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
+
+    const { isLoggedIn, isLoading } = useAuthenticated();
+
+    const { isLoading: isAuthenticating } = useGetUserQuery(skipToken);
+
+    // here FormatDate() is a function that is defined inside the util folder and
+    // is used to calculate currentDate - 1 which returns the date as May 16, 2023
+    // which is getting stored in the yesterdayDate variable
+    const yesterdayDate = FormatDate();
+
+    const handleFormSubmission = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        setButtonDisable(false);
+        setStandupUpdate({
+            type: 'user',
+            completed: '',
+            planned: '',
+            blockers: '',
+        });
+        //needed to set the button disabled=true so that it
+        // is clickable for the next time on the same day if user has
+        // to submit it
+        setButtonDisable(true);
+        console.log(standupUpdate);
+    };
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setStandupUpdate({
+            ...standupUpdate,
+            [event.target.name]: event.target.value,
+        });
+    };
+
+    return (
+        <Layout>
+            <Head title="Standup" />
+            {!isAuthenticating && isLoggedIn ? (
+                isLoading ? (
+                    <p>Loading...</p>
+                ) : (
+                    <StandUpContainer
+                        handleChange={handleChange}
+                        handleFormSubmission={handleFormSubmission}
+                        buttonDisable={buttonDisable}
+                        yesterdayDate={yesterdayDate}
+                        completed={standupUpdate.completed}
+                        blockers={standupUpdate.blockers}
+                        planned={standupUpdate.planned}
+                    />
+                )
+            ) : (
+                <div>
```

Comment:
Can you center this text?
![image](https://github.com/Real-Dev-Squad/website-status/assets/70854507/c11b73d5-59fb-42ff-8ba6-3d763b3c50fe)